RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeHitFilterESProducer_cfi.py

```diff
@@ -11,6 +11,5 @@
 )
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toModify(ClusterShapeHitFilterESProducer,
-    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase1Tk.par',
-    doPixelShapeCut = cms.bool(False)
+    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase2Tk.par',
```

Comment:
What if we check before the performance of T1 and T2 with this "wrong" calibration? In any case also the "phase1" ones were not optimal. I know that they are not the right ones for them, but if T3 is the only that we care for TDRs, and the performance on T1 and T2 are also kind of ok with the phase2Par, we do not need to create other eras or so which takes time and maybe confusion later.
I have started to produce results for [ttbar](http://ebrondol.web.cern.ch/ebrondol/Phase2Tracking/fixPhase2PixelGeometry/20161116_TTbar_D1/plots_ootb/effandfake1.pdf) in D1 geometry, with and without this PR. It does seem to me evetything all right.. @slava77 what do you think?